Makefile

```diff
@@ -76,6 +76,6 @@ update-binary-versions: force-update-versions $(KNOWN_BINARY_VERSIONS_FILES)
 .testdata/sample-binary-%: .testdata/binary-%
 	$(RM) $@
 	$(CAT) .testdata/stubheader-sample > $@
-	for prefix in $$($(SED_STRIP_COMMENTS) $< | $(GREP) -E '\.[0-9]\.|\.9' | $(CUT) -b1-3 | $(SORT) -r | $(UNIQ)) ; do \
+	for prefix in $$($(SED_STRIP_COMMENTS) $< | $(GREP) -E '\.[0-9][0-9]*(\.|$$)' | $(CUT) -b1-3 | $(SORT) -r | $(UNIQ)) ; do \
```

Comment:
Why not just use `+` instead of `*` so that `[0-9]` doesn't have to be duplicated?  ala `$(GREP) -E '\.[0-9]+(\.|$$)'`?

Reply:
Because I was half paying attention to a speaker on stage at this conference and wrote that bit on auto-pilot.  IOW: I was dumb.